gradlew

```diff
@@ -28,16 +44,16 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Are you sure we want to set the max JVM memory to 64mb?

Reply:
cc @JamieSinn probably knows things about this.

Reply:
that's just tying gradle's build call - it should be fine - but I wouldn't set max memory on anything java as a general best practice.

Reply:
I believe this is automatically set by the gradle version we are using